prow/plugins/plugins.go

```diff
@@ -217,6 +217,11 @@ type Owners struct {
 		---
 	*/
 	MDYAMLRepos []string `json:"mdyamlrepos,omitempty"`
+
+	// SkipCollaborators disables collaborator cross-checks and forces both
+	// the approve and lgtm plugins to use solely OWNERS files for access
+	// control in the provided repos.
+	SkipCollaborators []string `json:"skip_collaborators,omitempty"`
```

Comment:
Open to a better name for this.

Reply:
I like the current name.

I guess maybe I would paint the bike shed `OnlyOwners`? This way it specifies what we want (look at owners files) instead of what we don't want (look at collaborators).

Reply:
Keeping the current name, thanks for the ack